simulator/src/main/java/simblock/settings/SimulationConfiguration.java

```diff
@@ -35,9 +33,8 @@ public class SimulationConfiguration {
   /**
    * The consensus algorithm to be used.
    */
-  //TODO not documented in markdown
-  // TODO return to PoW
-  public static final String ALGO = "simblock.node.consensus.ProofOfWork";
+  // public static final String ALGO = "simblock.node.consensus.ProofOfWork";
```

Comment:
perhaps define an array of consts with these values and have an entry in the config with simpler values (pow, pospace) or get as a command-line parameter?